iset.mm

```diff
@@ -82997,6 +83073,100 @@ Nonnegative integers (as a subset of complex numbers)
     $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                              Extended nonnegative integers
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  The function values of the hash (set size) function are either nonnegative
+  integers or positive infinity.  To avoid the need to distinguish between
```

Comment:
"plus infinity" is used throughout

Reply:
There are 13 usages of "positive infinity" in set.mm. I don't object to changing them to "plus infinity", though, if that's the preference (it seems to be by number, as there are 45 of those).

Reply:
yes please, and also negative infinity --> minus infinity (in both set.mm and iset.mm, and not missing the ones with line breaks in the middle would be great)